modules/custom/moj_resources/src/SeriesContentApiClass.php

```diff
@@ -113,25 +115,22 @@ private function decorateSeries($node)
       $result["duration"] = $curr->field_moj_duration ? $curr->field_moj_duration->value : 0;
       $result["description"] = $curr->field_moj_description[0];
       $result["categories"] = $curr->field_moj_top_level_categories;
+      $result["prison_types"] = $curr->field_prison_types;
+
       if ($curr->field_moj_secondary_tags) {
         $result["secondary_tags"] = $curr->field_moj_secondary_tags;
       } else {
         $result["secondary_tags"] = $curr->field_moj_tags;
       }
-      $result["prisons"] = $curr->field_moj_prisons;
```

Comment:
We can probably remove this from all decorators anyways, I think content used it when it did the access checks on the frontend but that's now in the backend :)

Reply:
Yup, totally, we can remove this across the board as we update each endpoint